h2o-algos/src/main/java/hex/tree/SharedTreeModelWithContributions.java

```diff
@@ -30,20 +34,25 @@ public Frame scoreContributions(Frame frame, Key<Frame> destination_key) {
     return scoreContributions(frame, destination_key, null);
   }
 
-  @Override
-  public Frame scoreContributions(Frame frame, Key<Frame> destination_key, Job<Frame> j) {
-    if (_output.nclasses() > 2) {
-      throw new UnsupportedOperationException(
-              "Calculating contributions is currently not supported for multinomial models.");
-    }
-
+  private Frame cleanFrame(Frame frame) {
```

Comment:
the columns that a removed are sometimes called "special columns") - I would just call the method removeSpecialColumns

Reply:
Renamed